_site.yml

```
@@ -18,15 +18,18 @@ navbar:
     - icon: fa fa-rss
       href: index.xml
 base_url: https://blogs.rstudio.com/tensorflow/
-google_analytics: "UA-20375833-3"
 favicon: images/favicon.png
 collections:
   posts:
     creative_commons: CC BY
     disqus: tensorflow-for-r-blog
     share: [twitter, linkedin]
     subscribe: _subscribe.html
-output: distill::distill_article
+output: 
+  distill::distill_article:
+    includes:
+      in_header: |
+        <script defer data-domain="{YOUR DOMAIN},TBD" src="https://plausible.io/js/plausible.js"></script>  
```

You need to put the snippet code in a file 

* `plausible.html` 
```html
<script defer data-domain="{YOUR DOMAIN},TBD" src="https://plausible.io/js/plausible.js"></script>
````

and then 
```suggestion
output: 
  distill::distill_article:
    includes:
      in_header: plausible.html  
```